### PR TITLE
[ADD]Added a new module to set a Timesheet Billable Type to Non Billable forcefully

### PR DESCRIPTION
**Pull request by krunal267**

### CODE REVIEW

File: sale_timesheet_force_non_billable/models/account_analytic_line.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    def remove_non_billable(self):
+        for line in self:
+            line.force_non_billable = False
+        return True
+
+    @api.multi
+    def toggle_non_billable(self):
+        for line in self:
+            line.force_non_billable = not line.force_non_billable
+        return True
```
Comment (ivantodorovich) [inline sale_timesheet_force_non_billable/models/account_analytic_line.py]: You can avoid having to implement these actions with the boolean_toggle widget
```
widget="boolean_toggle"
```

It looks like this on tree views, without having to implement buttons ;)
image

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Thanks Krunal!

if you can replace the action with the boolean_toogle widget that'd be great.
It's simpler and it results in fewer lines of code which is always good for maintainability

Also lint is failing but it's not related to this module, so it's ok

File: sale_timesheet_force_non_billable/models/account_analytic_line.py (hunk 1 above)
Comment (krunal267) [reply sale_timesheet_force_non_billable/models/account_analytic_line.py]: Ohh... that's what I was looking for, but maybe I added bool field in button tag, so it wasn't visible to me

**Comment by ivantodorovich** [comment]
Aweome! :) I'm merging this

**Pull request merged**